### PR TITLE
README: Accepted RFC 37 - Custom Asset Claimer

### PR DESCRIPTION
**Pull request by mrshiposha**
An amendment to the README corresponding to the accepted RFC #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1050,6 +1050,15 @@ Kind: *Indication*
 Errors: 
 - `BadOrigin`
 
+### `SetAssetClaimer`
+
+Set a claimer to the dropped assets.
+
+The dropped assets are the assets that are still located in the `Holding Register` after a program finishes its execution.
+
+Kind: *Instruction*
```
Comment (muharem) [inline README.md:1059]: missing `Operands` section with the `MultiLocation`

Comment (mrshiposha) [reply README.md:1062]: Fixed

File: README.md [hunk 2]
```diff
@@ -1050,6 +1050,18 @@ Kind: *Indication*
 Errors: 
 - `BadOrigin`
 
+### `SetAssetClaimer`
+
+Set a claimer to the dropped assets.
```
Comment (KiChjang) [inline README.md:1055]: Is the term "dropped assets" used anywhere else in this document? If not, then I suggest simply calling it "assets in holding after execution finishes".

Comment (KiChjang) [reply README.md:1055]: Also, explaining the "claimer" term is also a good piece of text to have.

Comment (mrshiposha) [reply README.md]: Removed mention of the "dropped assets". Added the definition of a claimer to the description of the new instruction.

**Review by KiChjang — APPROVED** [review]

**Comment by muharem** [comment]
what is not clear to me from the process document, whether PR into the readme should be merged after the implementation of proposed changes or before.
https://github.com/paritytech/xcm-format/blob/838040fdfcab3a2ab95d7e32975bc8bc90fda5e9/proposals/0001-process.md?plain=1#L45

**Comment by KiChjang** [comment]
There isn't any formalized process for that yet, and it is still muddy for us. However, the spec is what we _want_ the XCVM to behave, and the `xcm-executor` crate is Parity's implementation of it. Strictly speaking, the two are completely separate concerns; it is not the job of the spec to ensure compliance, but rather engineers who maintain the implementation.

**Pull request merged**